sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/ClientMetricsTest.java

```diff
@@ -249,7 +262,7 @@ public void createItem() throws Exception {
                     this.validateMetrics(
                         expectedOperationTag,
                         expectedRequestTag,
-                        1,
+                        0,
```

Comment:
why the change here

Reply:
Random test flakiness - when backend indeed returns sometimes 0 RU.